recipes/imread/meta.yaml

```diff
@@ -0,0 +1,69 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "imread" %}
+{% set version = "0.6" %}
+{% set sha256 = "97364c44b9861ebd74dbf9a7cafea604a802543b3a9b463c9de68befed72d86c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - add_sys_prefix_include.patch
+
+build:
+  number: 0
+  # Patches for Windows compilation are welcome, currently fails to link
```

Comment:
Probably is that there's no `png.lib` in `png` windows conda package. You can copy `libpng.lib` to png.lib` so that it is found